### DISCUSSION TITLE
Adding coding tips to Roberts lab resources

### DISCUSSION THREAD

**Discussion by laurahspencer** [RobertsLab/resources#1092] [General]
Now that we have the new format for Roberts Lab resources, where should we add bits of code that we have found useful?  Also, do folks think tips should be screened by others, e.g. to make sure it's not poor advice?  For example ... 

I needed to get a large fasta file from a shared Google Drive to Mox. I found the following code to do so directly, rather than downloading to a computer and using `rsync`: 

`gdown -O PGA_assembly.fasta https://drive.google.com/uc?id=1Yanmb5yBXn-D4b_fzkR2GSxP` 

But, is this an "acceptable" way to transfer files to Mox?  

FYI for the above, the Google Drive file's ID is "1Yanmb5yBXn-D4b_fzkR2GSxP" and can be found within the link to share (NOTE, this example ID isn't real/valid; also this required that I install `gdown` on Mox).

**Comment by kubu4** [comment]
> where should we add bits of code that we have found useful

Somewhere under the "Computing and Code" section. You'll need/want to create a new markdown file (or, better yet, copy some of the various "tips and tricks" markdown files we have in the Resources/code repo and add this to one of the existing files). 

I'd also add the maybe we should prepend filenames with `code_` to help identify their contents more easily.

> But, is this an "acceptable" way to transfer files to Mox?

Yes, as long as you confirm the checksum matches that listed with the FastA file on Google Drive after you transfer it to Mox.

**Comment by sr320** [comment]
I went ahead and added as I code use this..
